spec/models/user_spec.rb

```diff
@@ -0,0 +1,16 @@
+require './app/data_mapper_setup'
+
+describe User do 
+	let!(:user) { User.create(email: 'ben@test.com', username: '@ben', password: 'password', password_confirmation: 'password') }
```

Comment:
Line is too long. [126/80]<br>Use 2 (not 1) spaces for indentation.<br>Tab detected.